cmd/kubeadm/app/phases/upgrade/postupgrade.go

```diff
@@ -61,7 +61,7 @@ func PerformPostUpgradeTasks(client clientset.Interface, cfg *kubeadmapi.InitCon
 	}
 
 	// Create the new, version-branched kubelet ComponentConfig ConfigMap
-	if err := kubeletphase.CreateConfigMap(&cfg.ClusterConfiguration, patchesDir, client); err != nil {
+	if err := kubeletphase.CreateConfigMap(&cfg.ClusterConfiguration, client); err != nil {
```

Comment:
with the original PR were we trying to fix a bug or a problem in e2e tests?
i'm trying to understand what implications does this have on upgrades too.
i guess on upgrade if the user passes --patches they would only apply on the local upgraded node, which is expected.
i.e. not apply the patch on the CM.

Reply:
I think yes, the upgrade also does not require applying these patches to the global ConfigMap.
/retest